### ISSUE TITLE
feature request: calculation functions...

### ISSUE THREAD

**Issue by edit4ever** [gutenberghub/gutenberg-forms#180]
It seems that the calculation field may only have basic math function.  Please add the functions of ceiling, floor and round - these are needed for more complicated equations.

**Comment by munirkamal** [comment]
Yes, there are basic calculations available for now that are needed for most common users. But, as this is an open-source project you are welcome to contribute to it.

**Comment by nikolaystrikhar** [comment]
Related to #261
